src/stdlib/String.ts

```diff
@@ -122,3 +122,50 @@ export const Mid = new Callable(
         return (n) ? new BrsString(s.value.substring(start, start + n.getValue())) : new BrsString(s.value.substring(start));
     }
 );
+
+/**
+ * Return a string from a float. If it is positive, prefix it with a space.  
+ */
+export const Str = new Callable(
+    {
+        name: "Str",
+        args: [{name: "value", type: ValueKind.Float}],
+        returns: ValueKind.String
+    },
+    (interpreter: Interpreter, value: Float): BrsString => {
+        const floatValue = value.getValue();
+        const prefix = (floatValue > 0.0) ? " " : "";
+        return new BrsString(prefix + String(floatValue));
+    }
+);
+
+/**
+ * Return a string from an integer. If it is positive, prefix it with a space.  
+ */
+export const StrI = new Callable(
+    {
+        name: "StrI",
+        args: [{name: "value", type: ValueKind.Int32}],
+        returns: ValueKind.String
+    },
+    (interpreter: Interpreter, value: Int32): BrsString => {
+        const intValue = value.getValue();
+        const prefix = (intValue > 0) ? " " : "";
+        return new BrsString(prefix + String(intValue));
+    }
+);
+
+/**
+ * Return a float from a string
+ */
+export const Val = new Callable(
+    {
+        name: "Val",
+        args: [{name: "s", type: ValueKind.String}],
+        returns: ValueKind.Float
+    },
+    (interpreter: Interpreter, s: BrsString): Float => {
+        return new Float(+s.value);
```

Comment:
This is fine, but consider using [Float.fromString](https://github.com/sjbarag/brs/blob/e5f4e01420a2e8145227152f314f06a8025026d8/src/brsTypes/Float.ts#L25-L34) or `new Float(Number(s.value))`  for future use-cases like this.  It's a bit more explicit than the `+` operator.

... though now that I think about it, should `Float.fromString` be changed to remove the `Number.parseFloat` call?  Seems unnecessary in light of the changes to the `Float` constructor in this PR. :thinking:

Reply:
I tried removing the parseFloat call in in Float, but that broke other tests, so I held off on that.  I did remove the '+' operator to be more explicit.

Reply:
Thanks for letting me know!